bin/git-release

```diff
@@ -34,6 +34,17 @@ if test $# -gt 0; then
     -c) need_changelog=true;;
     -r) remote=$2; shift ;;
     -m) msg=$2; shift ;;
+    -s)
+    test -n "$keyid" &&
```

Comment:
A little cavil: please indent the `-s)` block like the `--semver)` one below.

Reply:
done